client/sites/settings/site/visibility/index.jsx

```diff
@@ -138,21 +138,15 @@ const LaunchSite = () => {
 	if ( 0 === siteDomains.length ) {
 		querySiteDomainsComponent = <QuerySiteDomains siteId={ siteId } />;
 		btnComponent = <Button>{ btnText }</Button>;
-	} else if ( isPaidPlan && siteDomains.length > 1 ) {
+	} else {
 		btnComponent = (
 			<Button
 				onClick={ handleLaunchSiteClick }
 				busy={ isLaunchInProgress }
-				disabled={ ! isLaunchable || ( isDevelopmentSite && agencyLoading ) }
-			>
-				{ btnText }
-			</Button>
-		);
-		querySiteDomainsComponent = '';
-	} else {
-		btnComponent = (
-			<Button
-				href={ `/start/launch-site?siteSlug=${ siteSlug }&source=general-settings&new=${ site.title }&search=yes` }
+				disabled={
+					( isPaidPlan && siteDomains.length > 1 && ! isLaunchable ) ||
```

Comment:
I doubt the `!isLaunchable` case needs to be guarded by the `isPaidPlan && siteDomains.length > 1` check. But But I can see how it's keeping the logic as it was before.

Reply:
Yeah, I thought it was safest to keep the logic as is for now